test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -333,7 +334,43 @@ public async Task Initialize_InvalidFunctionNames_DoesNotCreateFunctionAndLogsFa
             }
         }
 
-        [Fact(Skip = "ApplyConfiguration no longer exists. Validate logic (moved to HostJsonFileConfigurationSource)")]
+        //[Fact(Skip = "ApplyConfiguration no longer exists. Validate logic (moved to HostJsonFileConfigurationSource)")]
+        //public void InitializeWorkers_Fails_AddsFunctionErrors()
```

Comment:
@fabiocav / @brettsam  - Can you please point me to the right place for this test?

Reply:
I'm moving those tests (actually splitting them, as the logic has moved to a couple of different services). If you need to update the verification, can  you create an issue for tracking?

Reply:
Created https://github.com/Azure/azure-functions-host/issues/3241 to add test